src/OutputImageParam.cpp

```diff
@@ -107,4 +107,10 @@ OutputImageParam &OutputImageParam::store_in(MemoryType type) {
     return *this;
 }
 
+Expr is_host_aligned(const OutputImageParam &param, const Expr &alignment_bytes) {
```

Comment:
(1) Why a standalone function instead of a method on OutputImageParam?
(2) Either way, this should be added to the Python bindings.